shared/js/background/email-utils.js

```diff
@@ -109,7 +109,7 @@ const getFullPixelName = (name, browserName) => {
     return `${name}_${browserName.toLowerCase()}`
 }
 
-const fireAddressUsedPixel = (pixel) => {
+const fireAutofillPixel = (pixel, shouldUpdateLastUsed = true) => {
```

Comment:
Wouldn't it be safer to pass `false` as default, in case we add more pixels in the future?